rss.go

```diff
@@ -87,6 +87,13 @@ type RssEnclosure struct {
 	Type    string   `xml:"type,attr"`
 }
 
+type RssGuid struct {
```

Comment:
The type should be named `RSSGUID` per [the Go style guide](https://google.github.io/styleguide/go/decisions#initialisms).  The `XMLAPI` example in the stye guide corresponds directly to this name because `RSS` and `GUID` are both initialisms.

Reply:
@canelohill I did notice that when I reviewed the PR but based on the fact that the repo already ignored it for other structs, I continued to ignore for now.